tests/test_vtk.py

```diff
@@ -45,6 +46,36 @@ def test_pluto_units(pluto_vtk_file):
         assert_allclose_units(getattr(ds, f"{u}_unit"), expected)
 
 
+def test_pluto_units_override(pluto_vtk_file):
+    file = pluto_vtk_file
+    ug = {
+        "time_unit": (2.0, "s"),
+        "length_unit": (4.0, "cm"),
+        "mass_unit": (5.0, "kg"),
+        "density_unit": (0.078125, "kg/cm**3"),
+        "velocity_unit": (2.0, "cm/s"),
+        "magnetic_unit": (62.66570686577499, "gauss"),
+    }
+    unit_combs = distinct_combinations(ug, 3)
+    invalid = yt_idefix.PlutoVtkDataset.invalid_unit_combinations
```

Comment:
please import the  PlutoVtkDataset class at the top level. It'll be easier to migrate this frontend to yt's main repo if every name changes are gathered at the top level of each files (or as much as possible)
```suggestion
    invalid = PlutoVtkDataset.invalid_unit_combinations
```

Reply:
It makes sense!